### DISCUSSION TITLE
Please tell me your favorite song!

### DISCUSSION THREAD

**Discussion by shota-in** [NAIST-SE/AttractingContributors#96] [General]
Please help me for doing my assignment!
Please tell me your favorite song.

This is my site. 
https://github.com/shota-in/AttractingContributors-Songs

**Comment by sergiodl-naist** [comment]
Added 能動的三分間 from Tokyo Incidents!
Please visit and try to ask a question or recommend me one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

**Reply by shota-in** [reply]
Thank you!
I want to ask your questions. but I can't push it for permission error.
Do you have any ideas?

This is the error message.
remote: Permission to sergiodl-naist/AttractingContributors-VTubers.git denied to shota-in.
fatal: unable to access 'https://github.com/sergiodl-naist/AttractingContributors-VTubers.git/': The requested URL returned error: 403

**Reply by shota-in** [reply]
I solved my problem and I added your list.

**Comment by zhangyongxin1212** [comment]
https://github.com/zhangyongxin1212/AttractingContributors-Game/pulls
added and plz help me
